pipelines/bootleg.py

```diff
@@ -82,15 +79,14 @@ def bootleg(
      .set_cpu_request('7.5')
      .set_cpu_limit('7.5')
      .set_gpu_limit(str(bootleg_num_gpus))
-     .add_volume_mount(V1VolumeMount(name='tensorboard', mount_path='/shared/tensorboard'))
      .add_volume_mount(V1VolumeMount(name='shm', mount_path='/dev/shm'))
      )
     (add_env(add_ssh_volume(bootleg_op), bootleg_env)
      .add_toleration(V1Toleration(key='nvidia.com/gpu', operator='Exists', effect='NoSchedule'))
      .add_node_selector_constraint('beta.kubernetes.io/instance-type', f'p3.{2 * bootleg_num_gpus}xlarge')
-     .add_volume(V1Volume(name='tensorboard',
-                          persistent_volume_claim=V1PersistentVolumeClaimVolumeSource('tensorboard-research-kf')))
     .add_volume(V1Volume(name='shm', empty_dir=V1EmptyDirVolumeSource(medium='Memory')))
      )
+
+    bootleg_op.container.set_image_pull_policy('Always')
```

Comment:
Do we need this? We don't, right?

Reply:
Not really but it's safer to have it. Pulling is fast compared to the job run time.

Reply:
Yes but there are other places where we don't specify it. This makes it sloppy, which is the opposite of safe. Better to fail early and often, than to fail later!

Reply:
We should add it to other pipelines too then. I really prefer to keep it but can remove if you insist..

Reply:
Yes, please remove it. It's recommended by Kubernetes authors not to use it.